lib/fuel-rest.js

```diff
@@ -130,6 +130,7 @@ FuelRest.prototype._makeRequest = function(consolidatedOpts, callback) {
 		// check if we should retry req
 		if(helpers.isValid401(res) && consolidatedOpts.retry) {
 			requestOptions.auth = consolidatedOpts.auth;
+			this.AuthClient.invalidateToken();
```

Comment:
Yeah, it had that. Just copied it and made a new test basically. I'll let you guys handle it. Jshint issue should be fixed as well